webpack.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   entry   : {
     genoverse: __dirname + '/index.js'
   },
-  target  : [ 'web', 'es5' ],
+  target  : [ 'node' ],
```

Comment:
Did you have to change this for a reason? Can we revert it to previous value?

Reply:
Reverted it. Don't remember why it was left like this. I guess I had issues with exporting Genoverse correctly